### ISSUE TITLE
Ctrl-Return should open context menu on Sequoia

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#4724]
https://lifehacker.com/tech/right-click-with-your-keyboard-in-mac-os-sequoia

This is now a native macOS feature, but it's broken everywhere in Zotero. I assume we'll need to add custom support for this to various components (editable-text, the trees, etc.).

We can create separate issues for individual components if that helps to track this.

**Comment by dstillman** [comment]
Ah, I was testing in an empty collection, so I didn't see the center right-click.

Could we fix this in individual components — at least some important ones — and detect the center right-click and ignore it? We could be a long ways from a Firefox fix. (Maybe we could backport something into 128, but who knows.)

**Comment by dstillman** [comment]
Excellent (and nice job finding the relevant WWDC session).

I assume we'd have to do some custom work here for the trees regardless. The focused element there is the whole tree.

**Comment by dstillman** [comment]
#4731 did almost everything, but a couple other places I noticed where this doesn't work:

1. Code editor (e.g., in Scaffold)
2. Annotations in the PDF reader — it's possible to get to the menu via the keyboard (or would be after https://github.com/zotero/zotero/issues/4732), but it would be quicker if this worked. This one is probably for @mrtcode.
